From d584241f47578691300c4589ff146115791d7826 Mon Sep 17 00:00:00 2001
From: Thomas Deutschmann <whissi@gentoo.org>
Date: Mon, 7 Sep 2020 18:16:41 +0200
Subject: [PATCH 3/4] build: Don't fail when passing standardized autotools
 options

We are not dealing with plain autotools build system. This will prevent
errors like

  mozbuild.configure.options.InvalidOptionError: Unknown option: --build

Signed-off-by: Thomas Deutschmann <whissi@gentoo.org>
---
 python/mozbuild/mozbuild/configure/__init__.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/python/mozbuild/mozbuild/configure/__init__.py b/python/mozbuild/mozbuild/configure/__init__.py
index c7cd96cdf5..19b74e7499 100644
--- a/python/mozbuild/mozbuild/configure/__init__.py
+++ b/python/mozbuild/mozbuild/configure/__init__.py
@@ -490,8 +490,8 @@ class ConfigureSandbox(dict):
             msg = 'Unknown option: %s' % without_value
             if self._help:
                 self._logger.warning(msg)
-            else:
-                raise InvalidOptionError(msg)
+            #else:
+            #    raise InvalidOptionError(msg)
 
         # Run the execution queue
         for func, args in self._execution_queue:
-- 
2.28.0

